src/bidiMapper/modules/cdp/CdpTarget.ts

```diff
@@ -521,7 +521,11 @@ export class CdpTarget {
 
   async toggleNetwork() {
     const stages = this.#networkStorage.getInterceptionStages(this.topLevelId);
-    const fetchEnable = Object.values(stages).some((value) => value);
+    const collectors = this.#networkStorage.getCollectorsForBrowsingContext(
+      this.topLevelId,
+    );
+    const fetchEnable =
```

Comment:
fetch does not need to be enabled for collectors to work, I think the change here needs to be reverted

Reply:
yes, removed this part.